docs/data-review-request.md

```diff
@@ -0,0 +1,105 @@
+
+# Request for data collection review form
+
+**All questions are mandatory. You must receive review from a data steward peer on your responses to these questions before shipping new data collection.**
+
+1) What questions will you answer with this data?
+
+TODO
```

Comment:
We will answer the following 5 questions

1. Is ‘within person’ effect  significant and  if any internet measurement is done in future do we need longitudinal measurements on profiles to control for their within - use variation. In other words, does the variability between a users day to day quality if internet experience contributes significantly to the over variability of internet experience.

2. Is 3 weeks of experiments enough time to collect 4 observations per profile (assuming they are not control nor on a slow connection) for 100% of profiles? We need multiple points to estimate variability

3. How many profiles on slow connections? We do not measure goodput for such profiles and if too many are on slow connections our data set will be biased.

4. Does usage remains unchanged for the profiles who will have downloads initiated in their idle background.